ThreeDSecure/src/main/java/com/braintreepayments/api/threedsecure/CardinalClient.java

```diff
@@ -78,27 +78,24 @@ private void configureCardinal(Context context, Configuration configuration,
         cardinalConfigurationParameters.setEnableDFSync(true);
 
         switch (request.getUiType()) {
-            case ThreeDSecureRequest.NATIVE:
-                cardinalConfigurationParameters.setUiType(CardinalUiType.NATIVE);
-            case ThreeDSecureRequest.HTML:
-                cardinalConfigurationParameters.setUiType(CardinalUiType.HTML);
-            case ThreeDSecureRequest.BOTH:
-                cardinalConfigurationParameters.setUiType(CardinalUiType.BOTH);
+            case NATIVE: cardinalConfigurationParameters.setUiType(CardinalUiType.NATIVE);
+            case HTML: cardinalConfigurationParameters.setUiType(CardinalUiType.HTML);
+            case BOTH: cardinalConfigurationParameters.setUiType(CardinalUiType.BOTH);
         }
 
         if (request.getRenderTypes() != null) {
             JSONArray renderTypes = new JSONArray();
 
-            for (Integer renderType : request.getRenderTypes()) {
-                if (renderType.equals(ThreeDSecureRequest.OTP)) {
+            for (ThreeDSecureRenderType renderType : request.getRenderTypes()) {
+                if (renderType.equals(ThreeDSecureRenderType.OTP)) {
```

Comment:
nit: we could technically `switch` here but the behavior would be the same.

Reply:
I tried utilizing a `switch` here, but it seems that Kotlin sealed classes don't play well with java `switch`es :( 

I'll be converting this entire class to Kotlin. So I'll be sure to convert this to a when statement.

Reply:
Oh cool yeah that makes sense 👍